lndclient/walletkit_client.go

```diff
@@ -47,6 +77,107 @@ func newWalletKitClient(conn *grpc.ClientConn,
 	}
 }
 
+// ListUnspent returns a list of all utxos spendable by the wallet with a number
+// of confirmations between the specified minimum and maximum.
+func (m *walletKitClient) ListUnspent(ctx context.Context, minConfs,
+	maxConfs uint32) ([]*lnwallet.Utxo, error) {
+
+	rpcCtx, cancel := context.WithTimeout(ctx, rpcTimeout)
+	defer cancel()
+
+	rpcCtx = m.walletKitMac.WithMacaroonAuth(rpcCtx)
+	resp, err := m.client.ListUnspent(rpcCtx, &walletrpc.ListUnspentRequest{
+		MinConfs: int32(minConfs),
+		MaxConfs: int32(maxConfs),
```

Comment:
This cast is problematic when the caller wants to pass in `math.MaxUint32` (after looking at the function signature).

Reply:
Yes, I noticed this issue as well on my end. Fixed.